Gruntfile.js

```diff
@@ -182,6 +182,19 @@ module.exports = function(grunt) {
 
     htmllint: {
       src: 'build/**/*.html'
+    },
+
+    pug: {
+      compile: {
+        options: {
+          data: {
+            debug: false
+          }
+        },
+        files: {
+          'build/blm.html': ['src/tmpl/blm.pug']
```

Comment:
There's already a task that picks up pages from templates. pages like plugins.html/pug and others. so you probably do not need this step or the extra plugin.

Reply:
This is odd, that's exactly what I thought, but for some reason the file wasn't being built so I added that plugin and task.